.github/workflows/bench.yml

```diff
@@ -45,6 +52,28 @@ jobs:
         cabal-version: ${{ matrix.cabal }}
         enable-stack: false
 
+    - name: Platform config
+      run: |
+        echo "CABAL_STORE_DIR=~/.cabal/store" >> $GITHUB_ENV
+        echo "CABAL_PKGS_DIR=~/.cabal/packages" >> $GITHUB_ENV
+
+    # Needs to be before Cache Cabal so the cache can detect changes to the modified cabal.project file
+    - if: matrix.ghc == '9.0.1'
+      name: (GHC 9.0.1) Use modified `cabal.project`
+      run: |
+        cp cabal-ghc901.project cabal.project
+
+    # Shorten binary names as a workaround for filepath length limits in Windows,
+    # but since tests are hardcoded on this workaround -
+    # all platforms (in 2021-12-07) need it.
+    - name: Workaround shorten binary names
```

Comment:
and this one, sorry to not be clearer in my first comment

Reply:
Well, ok.

It is strange to have this be literally required for `test`s, but not present in `bench`.

For me it is was surprising that running tests on Linux require this Windows workaround.

Reply:
in fact it is not needed, it could be under a condition